### PR TITLE
[#1794] Redesign case detail-page header

### PR DESCRIPTION
**Pull request by jiromaykin**
issue: https://taiga.maykinmedia.nl/project/open-inwoner/task/1793
The newly. designed Dashboard component requires restructuring of the detail page grid + has impact on Contactmoment detail page.
Also: Anchormenu references need to be removed for Case detail 
(and from Case list in other PR).

Note: unsure which variable is needed for "Verwachte uitslag" (now set to `end_date_legal`)

### CODE REVIEW

**Review by pi-sigma — CHANGES_REQUESTED** [review]
The spinner on the detail page is no longer centered after the removal of the sidebar anchors (see screenshot). Is this something that will be fixed in a separate PR?

Screenshot from 2023-10-26 11-01-32

**Comment by jiromaykin** [comment]
> The spinner on the detail page is no longer centered after the removal of the sidebar anchors"

I've fixed it in this PR for the Detail page only - but it should actually be simplified and homogenised in another PR for the List view after this is merged (because the sidebar is going to be removed in all Aanvragen pages), so I'll have remember that.

**Comment by pi-sigma** [comment]
Looking good (flaky tests but seems alright now).

**Comment by jiromaykin** [comment]
> Looking good (flaky tests but seems alright now).

Strangely enough the tests suddenly passed through this morning (after 3 days) - odd!

**Review by pi-sigma — APPROVED** [review]

File: src/open_inwoner/scss/components/Container/Container.scss [hunk 1]
```diff
@@ -14,7 +14,9 @@
   }
 
   @media (min-width: 768px) {
```
Comment (Bartvaderkin) [inline src/open_inwoner/scss/components/Container/Container.scss:16]: Are we sure about these magic numbers as breakpoints? I see `@media (min-width: 768px) {` here but in the file above there is `@media (min-width: 767px) {`

**Review by Bartvaderkin — APPROVED** [review]

File: src/open_inwoner/scss/components/Container/Container.scss (hunk 1 above)
Comment (jiromaykin) [reply src/open_inwoner/scss/components/Container/Container.scss:16]: Ah: **_min_**-width should be 768px everywhere, but **_max_**-width should be 767px

**Pull request merged**